LicenseHeaderManager/Utils/LinkedFileFilter.cs

```diff
@@ -14,6 +14,7 @@
 
 using System.Collections.Generic;
 using EnvDTE;
+using EnvDTE80;
```

Comment:
unused using. But as you seem to use that consistently, am i missing something?

Reply:
No, I think it just got added automatically as I was working on the PR. I'll clean these up.